misc/snapshotter/snapshotter.sh

```diff
@@ -154,3 +154,21 @@ function deploy_snapshotter() {
     configure_snapshotter
     ${COMMANDLINE} &
 }
+
+function cleanup_snapshotter() {
+    pid=$(ps -ef | grep containerd-nydus-grpc | grep -v grep | awk '{print $1}')
+    if [ ! -z "$pid" ]; then
+        for i in $(nsenter -t 1 -m ctr -n k8s.io snapshot --snapshotter nydus list | grep -v KEY | cut -d' ' -f1); do
+            nsenter -t 1 -m ctr -n k8s.io snapshot --snapshotter nydus rm $i || true
```

Comment:
I suppose this operation can fail if a running pod has used the snapshot. Here we ignore the fail and proceed to remove the other snapshots then finally remove nydus-snapshotter completely. I'm wondering what happens to the pod that is still using the snapshot. Does it keep running normally?